py/desispec/scripts/trace_shifts.py

```diff
@@ -274,10 +282,10 @@ def fit_trace_shifts(image,args) :
             if merr != 100000. :
                 log.warning("max edge shift error = %4.3f pixels is too large, reducing degrees"%merr)
 
-            if degxy>0 and degyy>0 and degxy>degxx and degyy>degyx : # first along wavelength
+            if degxy>0 or degyy>0 and degxy>degxx and degyy>degyx : # first along wavelength
```

Comment:
Please add parentheses here to make the order of operations of evaluating the `or` and `and`s very clear.

Reply:
Good catch. I checked the logic more carefully and realized I'd made a mistake; it should actually be
```
if (degxy>0 or degyy>0) and (degxy>degxx or degyy>degyx)
```
to match the intended behavior of the code.